packages/create-waku/src/index.ts

```diff
@@ -31,9 +30,12 @@ function canSafelyOverwrite(dir: string) {
 async function init() {
   let targetDir = '';
   const defaultProjectName = 'waku-project';
-
-  const templateRoot = fileURLToPath(new URL('../template', import.meta.url));
-  const CHOICES = await fsPromises.readdir(templateRoot);
+  const __dirname = dirname(fileURLToPath(import.meta.url));
+  const templateRoot = path.resolve(__dirname, '../template');
```

Comment:
Can we simplify like this?
```suggestion
  const templateRoot = path.join(fileURLToPath(import.meta.url), '../../template');
```